manifests/init.pp

```diff
@@ -233,6 +239,8 @@
   Integer[0] $api_service_worker_count = min(4, $facts['processors']['count']) + 1,
   Integer[0] $content_service_worker_timeout = 90,
   Integer[0] $api_service_worker_timeout = 90,
+  Integer[0] $api_service_worker_max_requests = 50,
```

Comment:
@evgeni We discovered the "real" root cause now, so we can relax this number significantly.  I would back it off to a much higher value, like say 1000 for the limit and 200 for the jitter.

Reply:
Do you have a link to info about the real root cause?

Reply:
See the discussion near the end on the BZ https://bugzilla.redhat.com/show_bug.cgi?id=2122872